tests/components/stream/conftest.py

```diff
@@ -58,3 +61,95 @@ def stream_worker_sync(hass):
         autospec=True,
     ):
         yield sync
+
+
+class HLSSync:
```

Comment:
This class allows us to sync the HLS provider across multiple requests as we feed in parts one by one.